build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env bash
+set -e
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
```

Comment:
Should this also use the root var?
```suggestion
MANIFEST_FILE=${SCRIPT_ROOT}/../../deploy/examples/multus-validation.yaml
```

Reply:
@travisn  wrote:
> Should this also use the root var?

this is meant to be a relative path. the root var is prepended further down for the `OUT_FILE` variable.